mocha-multi.js

```diff
@@ -124,7 +126,9 @@ function initReportersAndStreams(runner, setup, multi) {
 
     withReplacedStdout(stream, function() {
       var Reporter = resolveReporter(reporter);
-      var r = new Reporter(shim, options || {});
+      var r = new Reporter(shim, assign({}, multi.options, {
```

Comment:
The goal is here to behave exacly as what happens when a reporter is passed alone.
It will get the entrite `options` object from Mocha, where it can look for its custom options in a section `reporterOptions`.

In order to initialize multiple reporters, we provide still the same object, but replace `reporterOptions` for every reporter with its particular options.